### DISCUSSION TITLE
How to change order of pages?

### DISCUSSION THREAD

**Discussion by FactorD** [team-forms/community#22] [Q&A] [answered]
Hi everyone, 

I understand that there isn't a way to change the order of pages via the GUI yet. 

But we are also not able to find a way how to change the order via the source code either. 

We now have already created a five-page multi-form and need to add a landing page for it. 

Please assist! 

Best, 
Damon

**Comment by Erin-Dwyer** [comment, marked as answer]
Hi @FactorD,

Hopefully the ability to re-order pages in a wizard will be added soon. For now you can re-order them directly within the source code, please note these steps should really only be done by experienced users and I recommend making a copy of your source code before applying any changes.


1. In the top right hand corner of the form builder screen click the "..." icon followed by the " {} view source"
2. The source code of the form will appear in a code editor. By analysing the JSON you should find that each page of your form is represented within an array in the structure shown below.
<img width="1220" alt="image" src="https://github.com/team-forms/community/assets/154988548/ef7e7d01-1fda-4aa3-a9a3-542ccb624b59">
3. Move or copy all the code from one page before another page to re-arrange the pages.

![2024-04-29 11 12 24](https://github.com/team-forms/community/assets/154988548/d72f55b1-6d45-40f8-b9ba-e9e8dc2015df)

**Reply by FactorD** [reply]
Worked perfectly, thanks Erin!
